### PR TITLE
fix: Add OLM channel for minor versions

### PR DESCRIPTION
**Pull request by sayan-biswas**
Changes:
- Add OLM channel for minor versions, `openshift-builds-1.0` and `openshift-builds-1.1`

### CODE REVIEW

**Comment by adambkaplan** [comment]
Asking that we hold this pull request until #14 merges (and this PR is then rebased).

**Comment by sayan-biswas** [comment]
@adambkaplan Can this PR be merged? All the previous channels are also restored in this.

File: catalog/v4.12/openshift-builds-operator/catalog.yaml [hunk 1]
```diff
@@ -4,21 +4,54 @@ name: openshift-builds-operator
...
+  - name: openshift-builds-operator.v1.0.2
+    skips:
+      - openshift-builds-operator.v1.0.0
+      - openshift-builds-operator.v1.0.1
+  - name: openshift-builds-operator.v1.1.0
+    replaces: openshift-builds-operator.v1.0.2
+    skips:
+      - openshift-builds-operator.v1.0.0
+      - openshift-builds-operator.v1.0.1
+name: stable-v1
```
Comment (adambkaplan) [inline catalog/v4.12/openshift-builds-operator/catalog.yaml:34]: :+1: to restoring stable-v1. This PR should replace mine (#14)

File: catalog/v4.12/openshift-builds-operator/catalog.yaml [hunk 2]
```diff
@@ -4,21 +4,54 @@ name: openshift-builds-operator
...
+package: openshift-builds-operator
+schema: olm.channel
+---
+entries:
+  - name: openshift-builds-operator.v1.1.0
+name: openshift-builds-1.1
```
Comment (adambkaplan) [inline catalog/v4.12/openshift-builds-operator/catalog.yaml:40]: Per discussion in Slack, rename this to `builds-1.1`

```suggestion
name: builds-1.1
```

@sayan-biswas We need to add a release note that we are changing the channel naming convention to "builds-1.N" for z-stream only updates.

**Review by adambkaplan — CHANGES_REQUESTED** [review]
Name change for the v1.1 channel (suggestion needs to be copied to the other OCP versions).

Otherwise looks good - I'll close my competing PR.

File: catalog/v4.12/openshift-builds-operator/catalog.yaml (hunk 2 above)
Comment (sayan-biswas) [reply catalog/v4.12/openshift-builds-operator/catalog.yaml]: updated!

**Pull request merged**